restcomm/restcomm.interpreter/src/main/java/org/restcomm/connect/interpreter/VoiceInterpreter.java

```diff
@@ -2087,6 +2110,9 @@ public void execute(final Object message) throws Exception {
                 }
                 record(bridge);
             }
+            if(enable200OkDelay){
```

Comment:
@kacmos even thought with this LCM is working when enable200OKDelay == false, did you checked if LCM will work when the feature will be true?
I think LCM wont work when enable200OKDelay == true.
We need to find a better way to protect LCM